contents/blog/things-you-can-do-in-posthog.md

```diff
@@ -0,0 +1,210 @@
+---
+date: 2022-07-13
+title: 22 ways PostHog can help you build better products
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/product-people.png
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Guides", "Product analytics"]
+---
+
+PostHog is a growing platform.
+
+We used to call ourselves a product analytics platform, but product analytics is just one feature [among many](/product) these days.
+
+And that got us to wondering... just how many useful things can you do with PostHog?
+
+The answer? A lot – and this list is just a snapshot of the posibilities.
+
+### 1. Replace multiple services with just PostHog
+
+Let's start with a big one.
+
+You could use Amplitude for product analytics, Hotjar for session recording, Flagsmith for feature control, Segment as your customer data platform, Optimizely for testing, and Redshift for your data warehouse – these are sensible choices.
+
+Alternatively, you can just use PostHog.
```

Comment:
```suggestion
Alternatively, you may be able to just use PostHog.
```

Reply:
since we aren't at 100% parity across all these

Reply:
Is there any particular one we're weak on? I'd rather take one out and keep the stronger wording if we can.

Alternatively, I might tweak how we approach this.